### ISSUE TITLE
Notifications: Add Amazon SES to server

### ISSUE THREAD

**Issue by tenthirtyone** [unicef/bounties#43]
Requirements
1. Add support for `node-ses` to the server
2. Create a `sendEmail` function we can use to send notifications.

Examples:
https://github.com/unicef/juniper-portfolio/blob/develop/lib/email/index.js

**Comment by Naimaoye** [comment]
Hello @tenthirtyone , can I claim this?

**Comment by tenthirtyone** [comment]
@Naimaoye Yes. Just comment you are starting work, fork and begin work on a branch from your fork.

**Comment by sunny775** [comment]
Hi @mehranhydary . I am an outreacy applicant. I'd love to work on this

**Comment by Naimaoye** [comment]
I am working on this already @sunny775

**Comment by sunny775** [comment]
Got it @Naimaoye . But if you need any help, I can assist. I'm pretty much used to Amazon SES and nodemailer

**Comment by Naimaoye** [comment]
@tenthirtyone , can this PR be merged?
